tests/ballerina-test-integration/src/test/java/org/ballerinalang/test/observability/metrics/MetricsTestCase.java

```diff
@@ -0,0 +1,202 @@
+                "resource=\"getProduct\",service=\"metricsTest\",quantile=\"0.98\",}", regex);
+        expectedMetrics.put("http_response_time_seconds{http_method=\"GET\",http_url=\"/product\",protocol=\"http\"," +
+                "resource=\"getProduct\",service=\"metricsTest\",quantile=\"0.99\",}", regex);
+        expectedMetrics.put("http_response_time_seconds{http_method=\"GET\",http_url=\"/product\",protocol=\"http\"," +
+                "resource=\"getProduct\",service=\"metricsTest\",quantile=\"0.999\",}", regex);
+        expectedMetrics.put("http_response_time_seconds_count{http_method=\"GET\",http_url=\"/product\"," +
+                "protocol=\"http\",resource=\"getProduct\",service=\"metricsTest\",}", Pattern.compile("6.0"));
+        expectedMetrics.put("http_response_time_seconds_sum{http_method=\"GET\",http_url=\"/product\"," +
+                "protocol=\"http\",resource=\"getProduct\",service=\"metricsTest\",}", regex);
+        expectedMetrics.put("ballerina_scheduler_waiting_for_lock_worker_count", regex);
+        expectedMetrics.put("ballerina_sql:CallerActions_response_time_seconds_max{action=\"select\"," +
+                "db_instance=\"\",db_statement=\"SELECT * FROM Products\",db_type=\"sql\"," +
+                "peer_address=\"jdbc:hsqldb:hsql://localhost:9001/TEST_DB\",}", regex);
+        expectedMetrics.put("ballerina_sql:CallerActions_response_time_seconds{action=\"select\",db_instance=\"\"," +
+                "db_statement=\"SELECT * FROM Products\",db_type=\"sql\"," +
+                "peer_address=\"jdbc:hsqldb:hsql://localhost:9001/TEST_DB\",quantile=\"0.5\",}", regex);
+        expectedMetrics.put("ballerina_sql:CallerActions_response_time_seconds{action=\"select\",db_instance=\"\"," +
+                "db_statement=\"SELECT * FROM Products\",db_type=\"sql\"," +
+                "peer_address=\"jdbc:hsqldb:hsql://localhost:9001/TEST_DB\",quantile=\"0.75\",}", regex);
+        expectedMetrics.put("ballerina_sql:CallerActions_response_time_seconds{action=\"select\",db_instance=\"\"," +
+                "db_statement=\"SELECT * FROM Products\",db_type=\"sql\"," +
+                "peer_address=\"jdbc:hsqldb:hsql://localhost:9001/TEST_DB\",quantile=\"0.98\",}", regex);
+        expectedMetrics.put("ballerina_sql:CallerActions_response_time_seconds{action=\"select\",db_instance=\"\"," +
+                "db_statement=\"SELECT * FROM Products\",db_type=\"sql\"," +
+                "peer_address=\"jdbc:hsqldb:hsql://localhost:9001/TEST_DB\",quantile=\"0.99\",}", regex);
+        expectedMetrics.put("ballerina_sql:CallerActions_response_time_seconds{action=\"select\",db_instance=\"\"," +
+                "db_statement=\"SELECT * FROM Products\",db_type=\"sql\"," +
+                "peer_address=\"jdbc:hsqldb:hsql://localhost:9001/TEST_DB\",quantile=\"0.999\",}", regex);
+        expectedMetrics.put("ballerina_sql:CallerActions_response_time_seconds_count{action=\"select\"," +
```

Comment:
Is it the time spend on the query is represented by this metric? if so, how can you be certain that it's 6 seconds.

Reply:
It represents the count not the time